test/functional/pdo_sqlsrv/PDO102_MaxOutputParams.phpt

```diff
@@ -5,50 +5,34 @@ Fetch data as VARCHAR(MAX)
 --ENV--
 PHPT_EXEC=true
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require_once("skipif_mid-refactor.inc"); ?>
 --FILE--
 <?php
+require_once("MsCommon_mid-refactor.inc");
 
-include 'MsCommon.inc';
-
-function MaxOutputParamsTest($expected, $length)
+function maxOutputParamsTest($expected, $length)
 {
-    include 'MsSetup.inc';
-
     $outstr = null;
+    $conn = connect();
 
-    $conn = Connect();
-
-    CreateProc(
-        $conn,
-        "EXEC_TEST",
-        "@OUT varchar(80) output",
-        "SET NOCOUNT ON; select @OUT = '$expected'; return (0)
-        ");
+    $procName = "EXEC_TEST";
+    dropProc($conn, $procName);
+    $conn->query("CREATE PROC [$procName] (@OUT varchar(80) output) AS BEGIN
+                  SET NOCOUNT ON; select @OUT = '$expected'; return(0) END");
 
     $sql = "execute EXEC_TEST ?";
-
     $stmt = $conn->prepare($sql);
-
-    if ($length)
-    {   
-        $stmt->bindParam(1, $outstr, PDO::PARAM_STR, 10);
-    }
-    else
-    {
-        $stmt->bindParam(1, $outstr, PDO::PARAM_STR, 3);
-    }
-
+    $stmt->bindParam(1, $outstr, PDO::PARAM_STR, $length);
     $stmt->execute();
 
     echo "Expected: $expected Received: $outstr\n";
 
-    if ($outstr !== $expected)
-    {
+    if ($outstr !== $expected) {
         print_r($stmt->errorInfo());
+        dropProc($conn, $procName);
         return(-1);
     }
-
+    dropProc($conn, $procName);
     return(0);
 }
```

Comment:
Suggestion, set a flag to the right value, dropProc, then return that flag
Also, isn't returning true or false more intuitive than -1 or 0?